iteration/iteration_test.go

```diff
@@ -378,7 +371,7 @@ func (s *TestIterationRepository) TestLoadChildren() {
 		Name:       level2IterationName,
 		SpaceID:    space.ID,
 		Path:       parentPath,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c